pkg/api/grpcServer.go

```diff
@@ -155,14 +169,22 @@ func (s *GRPCServer) streamLoop(stream grpc.ServerStream, request proto.Message,
 			return err
 		}
 
+		var span ot.Span
+		span, ctx = ot.StartSpanFromContext(ctx, fmt.Sprintf("%s-%d", methodName, i))
+		span.LogFields(log.Object("gRPC request", request))
```

Comment:
How important do we think logging the fields will be?  I don't know that this matters immediately, but I imagine these requests can get quite large.  Should we make logging request and response fields optional (like a debug mode)? 

I think the real value will be in the spans themselves.